### PR TITLE
use go mod

### PR DESCRIPTION
**Pull request by pei0804**
**Describe the PR**
If use `go mod` We have to change to go 1.11 a minimum version. What do you think?
I thought contributor want to use same dependency so I wanto to quit use `go get`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,7 +13,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/stretchr/testify v1.2.2
+	github.com/stretchr/testify v1.3.0
```
Comment (pei0804) [inline go.mod]: `1.2.2` have some problem
https://travis-ci.org/swaggo/swag/jobs/508357216#L662

**Comment by pei0804** [comment]
@easonlin404 review plz.

File: .travis.yml [hunk 2]
```diff
@@ -2,20 +2,12 @@ language: go
 sudo: false
 
 go:
-  - 1.9.x
-  - 1.10.x
```
Comment (easonlin404) [inline .travis.yml]: I think that shouldn't delete against 1.9/1.0 testing because users might still be use older go version.

**Comment by pei0804** [comment]
I have one question.
1.9 and 1.10 doesn't have go mod.
Does They use `go get` ?
@easonlin404

File: .travis.yml (hunk 2 above)
Comment (pei0804) [reply .travis.yml]: Question for `use older go version.`
Old go can't use swag binary? or You say use case is contributor write swag code?

Comment (easonlin404) [reply .travis.yml]: user still can work if they still use go 1.9/1.10 version.

**Comment by easonlin404** [comment]
> I have one question.
> 1.9 and 1.10 doesn't have go mod.
> Does They use `go get` ?
> @easonlin404

users only can get package via using `go get` when go 1.9/1.10.

File: .travis.yml (hunk 2 above)
Comment (pei0804) [reply .travis.yml]: I understand.

**Comment by pei0804** [comment]
> users only can get package via using go get when go 1.9/1.10.

I wanted to match recognition for you.
Thank you. I create another PR.

BTW. User required use mod from 1.12x?

**Comment by easonlin404** [comment]
> > users only can get package via using go get when go 1.9/1.10.
> 
> I wanted to match recognition for you.
> Thank you. I create another PR.
> 
> BTW. User required use mod from 1.11x?

sure, you can reference https://github.com/golang/go/wiki/Modules docs.

**Pull request closed without merging**